src/components/recoil-test/RecoilInput.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { memo } from 'react';
+import { useRecoilState } from 'recoil';
+import { textState } from '@/store/sample';
+
+const RecoilInput: React.FC = () => {
+  const [text, setText] = useRecoilState(textState);
+
+  const onChange = (event: any) => {
```

Comment:
```suggestion
  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```
[May]
- 사소하지만 타입을 선언해두면 어떨까요?

Reply:
어떤 이벤트인지 몰라서, any로 잡았던..ㅠ 😢 
반영완료!!! 😄